azure-pipelines.yml

```diff
@@ -221,6 +222,87 @@ stages:
               summaryFileLocation: '$(System.DefaultWorkingDirectory)/coverage/coverage.xml'
               reportDirectory: '$(System.DefaultWorkingDirectory)/coverage/htmlcov'
 
+      - job: test_cloud_
```

Comment:
I broke cloud out to mirror what we do elsewhere since it hits an external dependency and can be flaky.